examples/windowmediaplayer.py

```diff
@@ -83,6 +80,22 @@ def windows_media():
     time.sleep(.5)
 
     app.ChooseColumsn.Cancel.click()
+    time.sleep(.5)
+
+    # Open a media file
+    app.WindowsMediaPlayer.menu_select("File->Open")
+
+    # Select and play a song
+    song_path = "#push your path"

```

Comment:
I'd suggest to use some standard sound from folder `C:\Windows\media` to make the script working out of the box.